jsettlers.common/src/jsettlers/common/menu/action/EActionType.java

```diff
@@ -116,6 +116,11 @@
 	SHOW_SELECTION,
 
 	/**
+	 * Sets the screen to display current message.
+	 */
+	GOTO_MSG,
```

Comment:
Yeah, I was wondering about `PAN_TO`, maybe that's why I chose a short name like this. But I'm glad you interject because I am very bad at coming up with names. `SHOW_MESSAGE` sure works for me.